### ISSUE TITLE
Text mode: title and content textarea focus style

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#1837]
In text mode, the title and content textarea elements should have a better focus style. Since text mode is supposed to be the most accessible one, and for some users will be the only mode that empowers them to enter content without barriers, I'm not sure the current behavior of the focus style is ideal.

when focus is on the title and the title is empty, the component gets an `is-selected` CSS class that applies a light gray outline `#e2e4e7` to the `h1` wrapping the textarea:

![screen shot 2017-07-10 at 15 44 00](https://user-images.githubusercontent.com/1682452/28021295-b17bda7c-6587-11e7-9f71-73b7cf903f89.png)

`#e2e4e7` doesn't have a sufficient color contrast with the background and when users start typing, the focus style disappears completely:

![screen shot 2017-07-10 at 15 44 05](https://user-images.githubusercontent.com/1682452/28021349-d7cc5e04-6587-11e7-94f2-b3fc5f996c5e.png)

The content textarea has no focus style at all:

![screen shot 2017-07-10 at 15 53 46](https://user-images.githubusercontent.com/1682452/28021403-0a35ef54-6588-11e7-8b20-76f492f4e7b7.png)

![screen shot 2017-07-10 at 15 45 30](https://user-images.githubusercontent.com/1682452/28021373-e7d6a49e-6587-11e7-8d85-274437e259a6.png)

**Comment by afercia** [comment]
Worth noting the post title is shared by visual and text mode. There are classes to target visual/text modes though.

Also, adding a focus style for the text mode content area would probably require to review a bit the textarea size/paddings. See in the screenshot below, where the red outline is just to highlight the current elements size:

![screen shot 2017-07-10 at 16 12 50](https://user-images.githubusercontent.com/1682452/28022506-ae915c16-658b-11e7-9c91-edac93ecb882.png)

**Comment by afercia** [comment]
Not sure why the `transition` property is set also on the state changes, as far as I know it should be set only on the initial state, see https://github.com/WordPress/gutenberg/blob/46ae03c851704a1087fd8a993ccf93dca8f423c6/editor/post-title/style.scss#L13-L29
/cc @mtias

**Comment by abdul** [comment]
@afercia:

1. What do you recommend for focus style on title/content in text-mode? I think, a thinner outline (like on on hover state) will do fine?
2. I think that's mistake "transition" property being set on state changes. 

Thank you

**Comment by afercia** [comment]
@abdul as long as it's a shape with a sufficient color contrast ratio with the background, it's ok for accessibility. As per the design part, I'll leave that decision to the team leads. /cc @jasmussen @mtias

**Comment by abdul** [comment]
@afercia Can you please review my PR?

**Comment by afercia** [comment]
@abdul as I said above, this needs design feedback from the design team before moving on 🙂

**Comment by abdul** [comment]
@afercia Please don't mind my restlessness. I will wait for feedback.

Thank you

**Comment by aduth** [comment]
Is this closed by #4803 ? cc @jasmussen

**Comment by jasmussen** [comment]
I would suggest yes. Closing, we can always reopen if there are edgecases that haven't been addressed.
